dotrix_math/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+pub mod math;
+
+pub type Mat4 = cgmath::Matrix4<f32>;
+pub type Point3 = cgmath::Point3<f32>;
+pub type Vec4 = cgmath::Vector4<f32>;
+pub type Vec3 = cgmath::Vector3<f32>;
+pub type Vec2 = cgmath::Vector2<f32>;
+pub type Quat = cgmath::Quaternion<f32>;
+
+#[cfg(test)]
```

Comment:
comment the test out with
// TODO: add real tests after removal of cgmath dependency